README.md

```diff
@@ -56,6 +64,51 @@ wrapStore(store, {portName: 'MY_APP'}); // make sure portName matches
 
 That's it! The dispatches called from UI component will find their way to the background page no problem. The new state from your background page will make sure to find its way back to the UI components.
 
+### 3. Optional: Implement actions whose logic only happens in the background script (we call them aliases)
+
+```js
+// background.js
+
+import { applyMiddleware, createStore } from 'redux';
+import { alias, wrapStore } from 'react-chrome-redux';
+
+const aliases = {
+  'user-clicked-alias': () => {
+    // this call can only be made in the background script
+    chrome.notifications.create(...);
+
+  };
+};
+
+const store = createStore(rootReducer,
+  applyMiddleware(
+    alias(aliases)
+  )
+);
+```
+
+```js
+// content.js
+
+import { Component } from 'react';
+
+window.store = ...; // a proxy store
```

Comment:
Just to limit confusion with assigning global window props, can we replace this with something like:

```js
const store = ...; // a proxy store
```